src/Symfony/Component/HttpKernel/Attribute/HttpException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Dejan Angelov <angelovdejan@protonmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class HttpException
```

Comment:
HttpStatus? This is not coupled to exceptions. It could be used on something else one day.
I would also nest this class in it's same-name namespace (like the others)

Reply:
Sorry @angelov (I'm the one who told him to move it up).